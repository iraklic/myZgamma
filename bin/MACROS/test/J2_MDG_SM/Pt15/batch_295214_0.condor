000 (295214.000.000) 11/07 10:26:10 Job submitted from host: <131.225.191.208:47381>
...
001 (295214.000.000) 11/07 10:26:49 Job executing on host: <131.225.206.242:47799>
...
006 (295214.000.000) 11/07 10:26:58 Image size of job updated: 81872
...
005 (295214.000.000) 11/07 23:43:06 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 13:15:09, Sys 0 00:00:13  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 13:15:09, Sys 0 00:00:13  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	127251  -  Run Bytes Sent By Job
	3008743  -  Run Bytes Received By Job
	127251  -  Total Bytes Sent By Job
	3008743  -  Total Bytes Received By Job
...
