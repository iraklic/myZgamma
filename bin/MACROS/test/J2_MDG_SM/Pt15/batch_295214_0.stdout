
CMSSW on Condor

started at Mon Nov  7 10:26:50 CST 2011

parameter set:
CONDOR_CLUSTER: 295214
CONDOR_PROCESS: 0
CMSSW_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_4_2_0
RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam
PARAMETER_SET: output_Mu.txt
NUM_EVENTS_PER_JOB: 

Writing final parameter-set: batch_295214_0.py to RUN_DIR: /uscms_data/d2/iraklich/Zgamma/CMSSW_3_8_4_1/src/ElectroWeakAnalysis/MultiBosons/bin/MACROS/test/myMinexam

run: myMinexam > batch_295214_0.log 2>&1\n

finished at Mon Nov  7 23:43:06 CST 2011
